### PR TITLE
I created the form view

### PR DESCRIPTION
**Pull request by hanna446**
I made some changes in the inputs and checkboxes; all rest is the same that mockup in trello.
form-quote
form-quote2

### CODE REVIEW

**Comment by hanna446** [comment]
[QUOTING FORM LAYOUT [2] - As a user i need a clear interface to allow me to select the products and services i would like to know prices on and possibly purchase](https://trello.com/c/TPtIbY6e/6-quoting-form-layout-2-as-a-user-i-need-a-clear-interface-to-allow-me-to-select-the-products-and-services-i-would-like-to-know-pr)

**Comment by hanna446** [comment]
[QUOTING FORM LAYOUT [2] - As a user i need a clear interface to allow me to select the products and services i would like to know prices on and possibly purchase](https://trello.com/c/TPtIbY6e/6-quoting-form-layout-2-as-a-user-i-need-a-clear-interface-to-allow-me-to-select-the-products-and-services-i-would-like-to-know-pr)

File: css/quotes.css [hunk 1]
```diff
@@ -1,4 +1,148 @@
-/* Quote page specific styles */
```
Comment (jusk02) [inline css/quotes.css:1]: Please make sure the file is indented at 2 spaces not 4.

File: js/quote_tool.js [hunk 2]
```diff
@@ -1 +1,11 @@
-// Qouting tool logic should be placed here.
\ No newline at end of file
+// Qouting tool logic should be placed here.
+const inputs = document.getElementsByClassName('input_form');
+for (let i = 0; i < inputs.length; i++) {
```
Comment (jusk02) [inline js/quote_tool.js]: Please document what you are doing here

File: pages/quote.html [hunk 3]
```diff
@@ -1,23 +1,123 @@
 <!DOCTYPE html>
 <html lang="en">
+
```
Comment (jusk02) [inline pages/quote.html]: please remove this empty line.

File: pages/quote.html [hunk 4]
```diff
@@ -1,23 +1,123 @@
...
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./../css/quotes.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
     <title>Cotizar - Hostal las paisas</title>
 </head>
+
```
Comment (jusk02) [inline pages/quote.html]: please remove this empty line

File: pages/quote.html [hunk 5]
```diff
@@ -1,23 +1,123 @@
...
+        </form>
+
     </section>
     <footer>
         <!-- footer section -->
     </footer>
+    <script src="./../js/quote_tool.js"></script>
 </body>
+
```
Comment (jusk02) [inline pages/quote.html]: please remove this empty line

File: pages/quote.html [hunk 6]
```diff
@@ -1,23 +1,123 @@
...
+                <label class="checkbox_container">
+                    <input type="checkbox" class="checkbox_dataUse">
+                    <span class="checkmark"></span>
+                    Tour 4
+                </label>
+            </div>
+
+
+        </form>
+
```
Comment (jusk02) [inline pages/quote.html]: please remove this empty line

File: pages/quote.html [hunk 7]
```diff
@@ -1,23 +1,123 @@
...
+                    <span class="checkmark"></span>
+                    Tour 3
+                </label>
+                <label class="checkbox_container">
+                    <input type="checkbox" class="checkbox_dataUse">
+                    <span class="checkmark"></span>
+                    Tour 4
+                </label>
+            </div>
+
```
Comment (jusk02) [inline pages/quote.html]: please remove these empty lines

File: .editorconfig [hunk 8]
```diff
@@ -0,0 +1,11 @@
+root = true
```
Comment (jusk02) [inline .editorconfig]: why did you change this file?

**Comment by jusk02** [comment]
Please add some screenshots to the PR as this is a purely visual chunk of code 😄

File: js/quote_tool.js [hunk 9]
```diff
@@ -1 +1,12 @@
-// Qouting tool logic should be placed here.
\ No newline at end of file
+//  this is a validation of labels; if in the inputs there are one or more characters,
```
Comment (jusk02) [inline js/quote_tool.js]: Please start with capita T, and mage sure spaces are the same on both lines.

also multi line commenst should go 
```
/**
 * First line
 * Second line
 */
```

File: css/quotes.css [hunk 10]
```diff
@@ -1,4 +1,180 @@
...
+* {
+    box-sizing: border-box;
```
Comment (camigomezdev) [inline css/quotes.css]: What does this do? Why do you apply this to everything?

File: css/quotes.css [hunk 11]
```diff
@@ -1,4 +1,180 @@
...
 
-/* We should use a mobile first approach having out breack point at 768px */
\ No newline at end of file
+.form {
+    margin-left: 5%;
+    margin-right: 5%;
+    max-width: 90%;
+}
+
+.form_title {
```
Comment (camigomezdev) [inline css/quotes.css]: Composed words should go with middle dash `.homepage-hero` and element children should go with a double low dash `.form__title`. This applies to all class name.

Reference: http://getbem.com/introduction/

File: css/quotes.css [hunk 12]
```diff
@@ -1,4 +1,180 @@
...
+    font-size: 22px;
+    font-weight: bold;
+    font-family: 'Roboto', sans-serif;
```
Comment (camigomezdev) [inline css/quotes.css]: The font-family should be in the `.form` class and not children elements.

File: css/quotes.css [hunk 13]
```diff
@@ -1,4 +1,180 @@
...
+hr {
+    border: 0;
+    margin-left: 0%;
+    margin-right: 50%;
+    width: 90px;
+    height: 1px;
+    background-color: #0086c3;
+}
+
+#hr_h3 {
```
Comment (camigomezdev) [inline css/quotes.css]: Please use a class instead of an id.

File: css/quotes.css [hunk 14]
```diff
@@ -1,4 +1,180 @@
...
+
+.form_subtitles {
+    font-weight: bold;
+    font-family: 'Roboto', sans-serif;
+    text-align: justify;
+    margin-bottom: 18px;
+    color: rgba(0, 0, 0, 0.7)
+}
+
+hr {
```
Comment (camigomezdev) [inline css/quotes.css]: Please use a class instead of a tag directly. This could affect other components.

File: css/quotes.css [hunk 15]
```diff
@@ -1,4 +1,180 @@
...
+    font-family: 'Roboto', sans-serif;
+    font-size: 12px;
+    font-weight: lighter;
+    margin-left: 5%;
+    margin-right: 5%;
+    text-align: justify;
+    margin-bottom: 40px;
+}
+
+.input_form:focus {
```
Comment (camigomezdev) [inline css/quotes.css]: Please add a comment describing how this works.

File: css/quotes.css [hunk 16]
```diff
@@ -1,4 +1,180 @@
...
+    outline: 1px rgba(0, 0, 0, 0.7);
+    border-bottom: 1px solid #0086c3;
+}
+
+.input_form:focus+.label_form {
```
Comment (camigomezdev) [inline css/quotes.css]: Please add a comment describing how this works.

File: css/quotes.css [hunk 17]
```diff
@@ -1,4 +1,180 @@
...
+    width: 17px;
+    height: 17px;
+    background: #eee;
+    position: absolute;
+    left: 0;
+    top: 0;
+    border-radius: 2px;
+}
+
+.checkbox_container input:checked+.checkmark {
```
Comment (camigomezdev) [inline css/quotes.css]: Please add a comment describing how this works.

File: css/quotes.css [hunk 18]
```diff
@@ -1,4 +1,180 @@
...
+    background-color: #0086c3;
+}
+
+.checkbox_container input:checked+.checkmark:after {
```
Comment (camigomezdev) [inline css/quotes.css]: Please add a comment describing how this works.

File: js/quote_tool.js [hunk 19]
```diff
@@ -1 +1,12 @@
...
+// the label should stay in the position assigned by the class "fix_tex".
+const inputs = document.getElementsByClassName('input_form');
```
Comment (camigomezdev) [inline js/quote_tool.js]: I think this should be a function.

File: css/quotes.css [hunk 20]
```diff
@@ -1,4 +1,193 @@
...
+    font-weight: lighter;
+    margin-left: 5%;
+    margin-right: 5%;
+    text-align: justify;
+    margin-bottom: 40px;
+}
+
+
+/*
+*In this code, i assign properties like border and color to the focused element.
```
Comment (camigomezdev) [inline css/quotes.css]: Please add comment between `*` and text.
Apply for all documentation. 👍

File: css/quotes.css [hunk 21]
```diff
@@ -1,4 +1,193 @@
...
+*/
+
+.form__input:focus {
```
Comment (camigomezdev) [inline css/quotes.css:53]: Please delete the space between Doc information and class name 😄 
Apply for all lines.

**Review by camigomezdev — APPROVED** [review]

**Review by Sonny97 — APPROVED** [review]

**Review by christian-cg — APPROVED** [review]
